components/brave_private_new_tab_ui/resources/page/private_new_tab.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html data-test-id="brave-new-tab-page" dir="$i18n{textdirection}">
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width">
+<title>Private New Tab</title>
+<link rel="stylesheet" href="chrome://brave-resources/css/reset.css">
+<link rel="stylesheet" href="chrome://resources/css/text_defaults.css">
+<link rel="stylesheet" href="chrome://brave-resources/fonts/poppins.css">
+<script src="chrome://resources/js/load_time_data.js"></script>
+<script src="/strings.js"></script>
+<style>
```

Comment:
I don't know whether we've thought about this yet, but in general it would be good to avoid inline scripts or inline styles so that we can use a strong CSP policy for these pages in the future.

So my recommendation here would be to move this to a `/styles.css` like the `/strings.js` we already have.